internal/constants/scripts.go

```diff
@@ -52,7 +52,7 @@ declare -A logical_nic_mac_map
 
 # Find destination directory based on ISO mode
 if [[ -f /etc/initrd-release ]]; then
-  ETC_NETWORK_MANAGER="/run/NetworkManager/system-connections"
+  ETC_NETWORK_MANAGER="/etc/coreos-firstboot-network"
 else
   ETC_NETWORK_MANAGER="/etc/NetworkManager/system-connections"
 fi
```

Comment:
I'm naive here as to how the assisted installer works. Seems like it would be nice if we could unify and just do it one way (i.e. always run in the initramfs). I'm sure there are great reasons for why it's done this way, but would like to learn the context if someone is willing to explain (happy to do a screenshare too).

Reply:
I can talk though this and all the history if you want to put something on my calendar

Reply:
The short answer to this though is that we're creating the placeholder for this initrd data when we create our minimal ISO (we're not currently using `coreos-installer` for any of this) so we don't have the ability to inject data into the initrd using the full ISO.